dev-qt/qtdeclarative/qtdeclarative-6.3.0.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit qt6-build
+
+DESCRIPTION="Qt Declarative (Quick 2)"
+
+if [[ ${QT6_BUILD_TYPE} == release ]]; then
+	KEYWORDS="~amd64"
+fi
+
+DEPEND="
+	=dev-qt/qtbase-${PV}*[gui,network,opengl,sql,test,widgets]
```

Comment:
a USE dependency on `[test]` seems very fishy to me

Reply:
oh but that's the testing library (QtTest), it should have nothing to do with USE=test, which controls the unit tests of qtbase itself

Reply:
Ah, right, that does make more sense.

I'll look more into it, thanks!

Reply:
It seems having testlib enabled will require network as well due to QuickControls2, if I'm looking at things right.  Going with unconditionally enabling network dependency for the time being.

Will look into the 'QT_FEATURE_testlib=OFF' build issue more when time allows.